docs/reference/balljointparameters.md

```diff
@@ -1,10 +1,16 @@
 ## BallJointParameters
 
+Derived from [JointParameters](jointparameters.md).
+
 ```
 BallJointParameters {
+  SFFloat position        0       # (-inf, inf)
   SFVec3f anchor          0 0 0   # any vector
+  SFFloat minStop         0       # [-pi, pi]
+  SFFloat maxStop         0       # [-pi, pi]
```

Comment:
These fields shouldn't be repeated here as they are inherited from JointParameters.